src/Stratis.FederatedPeg.Features.FederationGateway/SourceChain/MaturedBlocksProvider.cs

```diff
@@ -71,7 +71,7 @@ public async Task<List<IMaturedBlockDeposits>> GetMaturedDepositsAsync(int block
 
```

Comment:
in the code above, is there now a way to reuse the IDepositExtractor's GetNewlyMaturedBlock (just to try avoid duplicating the logic that goes back a few blocks) ?

Reply:
We're not really duplicating anything as one of those methods is calling  the other